src/pages/index.astro

```diff
@@ -3,192 +3,152 @@ import Main from '../layouts/Main.astro'
-                        researchers quickly up-and-running with SPSD/M, saving time and frustration.
-                    </p>
-                </div>
-                <div class="pillar" id="pillar3">
-                    <div class="pillar-box" id="blue">
-                        Data Science
-                    </div>
-                    <p>
-                        We develop processes for procuring, organizing, and analyzing information effectively and
-                        efficiently. We also provide our clients with analysis they can easily replicate such as annual
-                        factsheets or reports.
-                    </p>
-                </div>
-            </div>
-            <OurWork />
-        </div>
-    </div>
+	<section id="hero">
+		<h1>
+			Today. Tomorrow. Together.
+		</h1>
+		<p>
+			Vivic Research is an economic consulting firm dedicated to empowering
+			peers working toward social justice with data-driven research.
+		</p>
+	</section>
+	<section id="services">
+		<h2>
+			Our Services
+		</h2>
+		<div class="pillars">
+			<section>
+				<h3>
+					Economic Research
+				</h3>
+				<p>
+					We develop boutique research studies focusing on targeted issues
+					related to the intersection of policy, economics, and social justice.
+					This includes but is not limited to literature reviews, policy
+					recommendations, community consultation and cost/benefit analysis.
+					Our past work has included costing an alternative municipal budget
+					for the city of Ottawa with an emphasis on racial and gender equity,
+					research on Employment Insurance and best practices for post-secondary
+					education fees and debt cancellation.
+				</p>
+			</section>
+			<section>
+				<h3>
+					Training
+				</h3>
+				<p>
+					We provide personalized training on quantitative research methods
+					for teams, helping them build confidence and do their work more
+					efficiently. Vivic offers SPSD/M training for beginners and more
+					experienced users. Our introductory training "Getting Started:
+					Policy Analysis Using SPSD/M" gets researchers quickly up-and-running
+					with SPSD/M, saving time and frustration.
+				</p>
+			</section>
+			<section>
+				<h3>
+					Data Science
+				</h3>
+				<p>
+					We develop processes for procuring, organizing, and analyzing
+					information effectively and efficiently. We also provide our
+					clients with analysis they can easily replicate such as annual
+					factsheets or reports.
+				</p>
+			</section>
+		</div>
+		<OurWork/>
+	</section>
 </Main>
 
 <style>
-
-#maincontent {
-    font-family: var(--mainfont);
-    margin: 325px 0 0;
-}
-
 #services {
-    max-width: 1500px;
-    margin: 0 auto 75px auto;
+	max-width: 1500px;
+	margin: 0 auto 75px auto;
 }
 
 #hero {
-    margin: 80px 8% 155px 8%;
-    font-family: var(--mainfont);
```

Comment:
Dropped all the `font-family: var(--mainfont);`s here, since that's already assumed by the main layout.